strategy/sim2.py

```diff
@@ -30,10 +31,36 @@ def __init__(self, name, team, world, speed, location):
         self.speed = speed
         self.location = location
         self.has_cube = True
-
+    def Robot1(drive):
+        Robot.name = 'Robot#1'
+        Robot1.team = BLUE
+        Robot1.speed = 5
+        Robot1.location = BLUE_PORTAL
+        Robot1.has_cube = True
+        
+        
+
+        if has_cube == False:
+            Robot1.pick_up
+        else:     
+            Robot1.put_down
     def __lt__(self, other):
         return hash(self) < hash(other)
-
+    def world():
```

Comment:
We discussed making `World` a class, not just a function, so it could work like all the other robots.